server/tutvwebsite/settings/base.py

```diff
@@ -100,10 +101,10 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/2.2/howto/static-files/
 
+
 STATICFILES_DIRS = [
-    os.path.join(BASE_DIR, "../build"),
+    os.path.join(BASE_DIR, "../build/static"),
 ]
 
-WHITENOISE_INDEX_FILE = True
-
-django_heroku.settings(locals())
+STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
```

Comment:
It might be good to rename "static" and "staticfiles" to something more descriptive of what their separate purposes are - right now I'm not sure that if I were looking for a certain file, I'd be able to guess whether it lived in "static" or "staticfiles" without reading through a lot of the code.